.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -2,4 +2,4 @@
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
   </state>
-</component>
+</component>
```

Comment:
I'd like to keep new line at the end of file

Reply:
Didn't mean to change this, IntelliJ keeps updating this file for some reason.